bin/oref0-ed-battery.sh

```diff
@@ -2,7 +2,6 @@
 
 echo Run oref0-ed-battery
 if ! /home/edison/src/EdisonVoltage/voltage json batteryVoltage battery | grep "-66" 2>/dev/null; then
-    echo start Voltage server and trigger
-    cd /home/edison/src/EdisonVoltage && sudo ./voltage_server &
-    cd /home/edison/src/EdisonVoltage && echo 1 > /tmp/battery_trigger
+    echo start Voltage
+    sudo /home/edison/src/EdisonVoltage/voltage
```

Comment:
Calling this here doesn't look necessary. You've already called it in the `if` condition (and then directed the output to NULL). What is this block of code supposed to do?